tools/rosbag/package.xml

```diff
@@ -31,6 +31,7 @@
   <run_depend>genmsg</run_depend>
   <run_depend>genpy</run_depend>
   <run_depend>python-crypto</run_depend>
+  <run_depend>python-pycryptodome</run_depend>
```

Comment:
There needs to be a `rosdep` key created for this: https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml

Reply:
https://github.com/ros/rosdistro/pull/20128